### DISCUSSION TITLE
How to change the font of the project cards and make the cards have same size?

### DISCUSSION THREAD

**Discussion by xinjie-liu** [alshedivat/al-folio#785] [Q&A] [answered]
Thanks for the great theme!

I am trying to align the project cards for my website. But I am new to website development, I do not know how to change the font size on the cards to be smaller and set the card size to be the same for all the projects. I've read the previous issues and discussion and tried to change the parameters in the file `_base.scss` as well as other `.html` files, but did not manage to do so. Perhaps could someone help me?

Thank you!!

**Comment by souravcipher** [comment]
Hi @xinjie-liu, did you figure out how to do it? 
What files did you change?

**Comment by george-gca** [comment]
To change the font size you could set the font size in the related css class used, in the case, [card-title](https://github.com/alshedivat/al-folio/blob/bbbb1394f24fa73dd687f48428553742b0aece53/_sass/_base.scss#L93). Regarding the cards sizes, what keep changing their size?

**Comment by KaroBe** [comment, marked as answer]
I was wondering about a similar thing, and these are some settings and changes I found useful:

**1. Horizontal**
In `projects.md` you can change the value to `horizontal: true`, which will result in image and text/description being arranged side-by-side.

**2. One column**
We see in the `{% if page.horizontal -%}` condition that `projects_horizontal.html` will be included for each project:

```
<div class="row row-cols-2">
    {%- for project in sorted_projects -%}
      {% include projects_horizontal.html %}
    {%- endfor %}
</div>
```

The card items are arranged in two rows. I didn't like that, because it cause very narrrow cards in the mobile format. Instead I replaced with: `<div class="row row-cols-1">`

**3. Font size of card titles**
> To change the font size you could set the font size in the related css class used, in the case, [card-title](https://github.com/alshedivat/al-folio/blob/bbbb1394f24fa73dd687f48428553742b0aece53/_sass/_base.scss#L93).

Specifically, make the change in `_base.scss`:
```
.card {
  ...
  .card-title {
    color: var(--global-text-color);
    font-size: 2rem;
  }
```
@xinjie-liu solved it by changing the html to a `<h6>` instead of `<h3 class="card-title">{{ project.title }}</h3>`.

**3. just for fun**
Another thing I did, in `projects_horizontal.html` remove text_lowecase from the title of the card because project titles often are proper nouns.
`<h3 class="card-title text-lowercase">{{ project.title }}</h3>`


**Lastly** I disabled masonry in `_config.yml` but I'm not sure whether that had an effect in the horizontal mode and don't want try and figure it out right now...
```
enable_masonry:             false   # enables automatic project cards arrangement
```

Hope this will help someone :)

**Reply by GENGMinghong** [reply]
Thanks! I followed your suggestion to make the card arrangement more elegant by horizontalizing them and disabling automatic lowercase titles.
